sections/examples/dockerfile/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:14.8.0-alpine AS build
+
+# Copy dependency information and install all dependencies
+COPY --chown=node:node package.json package-lock.json ./
+
+RUN npm ci
+
+# Copy source code (and all other relevant files)
+COPY --chown=node:node src ./src
+
+# Build code
+RUN npm run build
+
+# Run-time stage
+FROM node:14.8.0-alpine
```

Comment:
Show installation of private packages by using --secret mount

Reply:
I started doing it but it wasn't easy to understand what was happening without the `docker build` command alongside. So I skipped it for now. Let's do it when we create a full example app and add a ci step